src/Binance/Spot/API/AccountTrade.jl

```diff
@@ -0,0 +1,115 @@
+module AccountTrade
+
+export AccountTradeQuery,
+    AccountTradeData,
+    account_trade
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Binance
+using CryptoAPIs: Maybe, APIsRequest
+
+Base.@kwdef mutable struct AccountTradeQuery <: BinancePrivateQuery
+    symbol::String
+    orderId::Maybe{Int64} = nothing
+    startTime::Maybe{DateTime} = nothing
+    endTime::Maybe{DateTime} = nothing
+    fromId::Maybe{Int64} = nothing
+    limit::Maybe{Int64} = 1000
+
+    recvWindow::Maybe{Int64} = nothing
+    timestamp::Maybe{DateTime} = nothing
+    signature::Maybe{String} = nothing
+end
+
+struct AccountTradeData <: BinanceData
+    symbol::String
+    id::Int64
+    orderId::Int64
+    orderListId::Int64
+    price::Float64
+    qty::Float64
+    quoteQty::Float64
+    commission::Float64
+    commissionAsset::String
+    time::NanoDate
+    isBuyer::Bool
+    isMaker::Bool
+    isBestMatch::Bool
+end
+
+"""
+    account_trade(client::BinanceClient, query::AccountTradeQuery)
+    account_trade(client::BinanceClient; kw...)
+
+Get trades for a specific account and symbol.
+
+[`GET api/v3/myTrades`](https://binance-docs.github.io/apidocs/spot/en/#account-trade-list-user_data)
+
+## Parameters:
+
+| Parameter  | Type     | Required | Description   |
+|:-----------|:---------|:---------|:--------------|
+| symbol     | String   | true     |               |
+| orderId    | Int64    | false    |               |
+| startTime  | DateTime | false    |               |
+| endTime    | DateTime | false    |               |
+| fromId     | Int64    | false    |               |
+| limit      | Int64    | false    | Default: 1000 |
+| recvWindow | Int64    | false    |               |
+| signature  | String   | false    |               |
+| timestamp  | DateTime | false    |               |
+
+## Code samples:
+
+```julia
+using Serde
+using CryptoAPIs.Binance
+
+binance_client = BinanceClient(;
+    base_url = "https://api.binance.com",
+    public_key = ENV["BINANCE_PUBLIC_KEY"],
+    secret_key = ENV["BINANCE_SECRET_KEY"],
+)
+
+result = Binance.Spot.account_trade(
+    binance_client;
+    symbol = "BTCUSDT",
+)
+
+to_pretty_json(result.result)
+```
+
+## Result:
+
+```json
+[
+  {
+    "symbol": "BNBBTC",
+    "id": 28457,
+    "orderId": 100234,
+    "orderListId": -1,
+    "price": "4.00000100",
+    "qty": "12.00000000",
+    "quoteQty": "48.000012",
+    "commission": "10.10000000",
+    "commissionAsset": "BNB",
+    "time": 1499865549590,
+    "isBuyer": true,
+    "isMaker": false,
+    "isBestMatch": true
+  },
+  ...
+]
```

Comment:
```suggestion
[
  {
    "symbol":"BNBBTC",
    "id":28457,
    "orderId":100234,
    "orderListId":-1,
    "price":4.00000100,
    "qty":12.00000000,
    "quoteQty":48.000012,
    "commission":10.10000000,
    "commissionAsset":"BNB",
    "time":"2017-07-12T13:19:09",
    "isBuyer":true,
    "isMaker":false,
    "isBestMatch":true
  },
  ...
]
```

If there is no real answer for the method, then let's at least formalize the answer given in the Binance documentation to our own data types